src/stdlib.js

```diff
@@ -11,6 +11,27 @@ export const lightscriptImports = {
   "bitwiseZeroFillRightShift": "inline",
 };
 
+export const runtimeHelpers = {
```

Comment:
ah, yeah, that's why. I kind of don't mind – it's babel helping make sure that I'm writing cross-platform code. But yes, if you see the tests, the output isn't the exact same as what I wrote.